Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -11,10 +11,14 @@ public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
 /// An `UIView` subclass that shows summary and preview information about a given channel.
 open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider, SwiftUIRepresentable {
     /// The data this view component shows.
-    public var content: (channel: _ChatChannel<ExtraData>?, currentUserId: UserId?) {
```

Comment:
I wonder what's the policy in terms of breaking changes in API's. I believe there is a best practice that you can only break compilation when you release a major version, but ChatUI is not really in prod yet, right?

Reply:
The public API is currently available only in beta, so it's OK to change it.